custom/icds_reports/ucr/data_sources/household_cases.json

```diff
@@ -90,6 +90,88 @@
         "type": "raw",
         "property_name": "closed_on"
       },
+      {
+        "datatype": "string",
+        "is_primary_key": false,
```

Comment:
you dont need `is_primary_key` and `is_nullable` key to add in every field.

Reply:
just added them from earlier columns..will remove them

Reply:
removed